Mathlib/CategoryTheory/Adhesive.lean

```diff
@@ -93,7 +93,10 @@ theorem IsPushout.isVanKampen_iff (H : IsPushout f g h i) :
       · dsimp; simp only [Functor.map_id, Category.comp_id, Category.id_comp]
       exacts [hf.w, hg.w]
     · ext (_ | _ | _)
-      · dsimp; rw [PushoutCocone.condition_zero]; erw [Category.assoc, hh.w, hf.w_assoc]
+      · dsimp
+        rw [PushoutCocone.condition_zero, Category.assoc]
+        erw [hh.w]
+        rw [hf.w_assoc]
```

Comment:
I would like to move to only using a single lemma in `erw` throughout the library. This both makes counting easier, and allows use of `erw?`.